app/javascript/components/GeneralInfrastructureMapping/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Field, reduxForm } from 'redux-form';
+import { renderField } from '../../common/formUtilities/renderField';
+import { required } from '../../common/formUtilities/validation';
+
+const GeneralInfrastructureMappingContainer = props => (
+  <form className="form-horizontal">
```

Comment:
Does it make sense to have a form component? Maybe in pf-react?

Reply:
I believe we already have a `Form` component in pf-react - I could use that.